installer/x86_64/install.sh

```diff
@@ -163,11 +163,21 @@ create_demo_gpt_partition()
         while read -r part_index; do
             if [ "$blk_dev$part_index" = "$cur_part" ]; then continue; fi
             echo "deleting partition $part_index ..."
+            # if the partition is already mounted, umount first
+            df $blk_dev$part_index 2>/dev/null && {
```

Comment:
is it better to use grep $blk_dev$part_index /proc/mount

Reply:
It's functionally equal. The benefit is easier code, no tricky string matching.